### PR TITLE
Do not show bundles in premium feature widgets

### PR DESCRIPTION
**Pull request by tsteur**
In "Latest Marketplace Updates" and "Premium Features" widgets which are on the admin home page, do not show bundles as it seems less confusing etc and I think it is not needed to show them there. Better to show individual features.

### CODE REVIEW

File: plugins/Marketplace/templates/getPremiumFeatures.twig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="getNewPlugins getPremiumFeatures widgetBody">
     <div class="row">
-        {% for plugin in plugins %}
+        {% for plugin in plugins if not plugin.isBundle %}
```
Comment (mneudert) [inline plugins/Marketplace/templates/getPremiumFeatures.twig]: Is there a special case preventing you from using the same `array_filter` like the plugin listing? Filtering the entries here seems to add an unintuitive extra limit to the already max 20 entries.

Comment (tsteur) [reply plugins/Marketplace/templates/getPremiumFeatures.twig]: good point, will update PR

Comment (tsteur) [reply plugins/Marketplace/templates/getPremiumFeatures.twig]: Updated :)

**Review by mneudert — APPROVED** [review]
LGTM.

Btw the `empty($plugins)` check in `GetPremiumFeatures.php` could also be removed. It probably was not needed for a long time as `searchForPlugins` should always return an array.

(Tests should of course still work before merging. The current failures however seem unrelated to this specific change)

**Pull request merged**